### PR TITLE
WT-4990 Migrate Jenkins wiredtiger-test-unit-ppc job to Evergreen

### PR DESCRIPTION
**Pull request by raviprakashgiri29**
Created a new buildvariant to support "ubuntu1804-power8-test" machine and running the unit-test on it.

### CODE REVIEW

**Comment by raviprakashgiri29** [comment]
Evergreen patch link below
https://evergreen.mongodb.com/version/5dcb4d7230661501c3dc97fc

File: test/evergreen.yml [hunk 1]
```diff
@@ -1735,3 +1735,17 @@ buildvariants:
...
+  - ubuntu1804-power8-test
+  expansions:
+    configure_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-ggdb -fPIC"
+    smp_command: -j $(grep -c ^processor /proc/cpuinfo)
+    posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-python --enable-zlib --enable-snappy --enable-strict --enable-static --prefix=$(pwd)/LOCAL_INSTALL
+    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
+    make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
+  tasks:
+  - name: compile
+  - name: unit-test
```
Comment (quchenhao) [inline test/evergreen.yml]: Add a new line at the end of the file

Comment (raviprakashgiri29) [reply test/evergreen.yml]: Fixed

**Comment by raviprakashgiri29** [comment]
retest this please

**Comment by raviprakashgiri29** [comment]
evergreen retry

File: test/evergreen.yml [hunk 2]
```diff
@@ -1735,3 +1735,17 @@ buildvariants:
   - name: generate-datafile-big-endian
   - name: verify-datafile-big-endian
   - name: verify-datafile-from-little-endian
+
+- name: ubuntu1804-powerpc
+  display_name: Ubuntu 18.04 PowerPC
```
Comment (lukech) [inline test/evergreen.yml]: I'd prefer using the well-known alias name "ppc/PPC" :)

Comment (raviprakashgiri29) [reply test/evergreen.yml]: Updated the task and display name to ppc/PPC .

**Pull request merged**